bimmer_connected/vehicle.py

```diff
@@ -149,14 +149,13 @@ def drive_train_attributes(self) -> List[str]:
         result = ['remaining_range_total', 'remaining_fuel', 'mileage']
         if self.has_hv_battery:
             result += ['charging_time_remaining', 'charging_status', 'max_range_electric', 'charging_level_hv',
-                       'chargingConnectionType', 'chargingInductivePositioning', 'connectionStatus',
-                       'lastChargingEndReason', 'remaining_range_electric', 'lastChargingEndResult']
+                       'chargingConnectionType', 'connectionStatus', 'lastChargingEndReason',
+                       'remaining_range_electric', 'lastChargingEndResult']
         if self.has_internal_combustion_engine:
             result += ['remaining_range_fuel']
         if self.has_hv_battery and self.has_range_extender:
             result += ['maxFuel', 'remaining_range_fuel']
         if self.has_hv_battery and self.has_internal_combustion_engine:
-            result += ['fuelPercent']
             result.remove('max_range_electric')
```

Comment:
Do we need to remove `max_range_electric`? As my vehicle has a battery and internal combustion engine, but still provides this key.

<details>
  <summary>Status response</summary>

  ```json
{
  "vehicleStatus": {
    "DCS_CCH_Activation": "NA",
    "DCS_CCH_Ongoing": false,
    "cbsData": [
      {
        "cbsDescription": "Next service due when the stated distance has been covered or by the specified date.",
        "cbsDueDate": "2022-09",
        "cbsRemainingMileage": 22000,
        "cbsState": "OK",
        "cbsType": "OIL"
      },
      {
        "cbsDescription": "Next statutory vehicle inspection due by the stated date.",
        "cbsDueDate": "2021-06",
        "cbsState": "OK",
        "cbsType": "VEHICLE_TUV"
      },
      {
        "cbsDescription": "Next visual inspection due when the stated distance has been covered or by the stated date.",
        "cbsDueDate": "2024-09",
        "cbsRemainingMileage": 60000,
        "cbsState": "OK",
        "cbsType": "VEHICLE_CHECK"
      },
      {
        "cbsDescription": "Next change due at the latest by the stated date.",
        "cbsDueDate": "2022-06",
        "cbsState": "OK",
        "cbsType": "BRAKE_FLUID"
      }
    ],
    "chargingConnectionType": "CONDUCTIVE",
    "chargingLevelHv": 47,
    "chargingStatus": "INVALID",
    "checkControlMessages": [],
    "connectionStatus": "DISCONNECTED",
    "doorDriverFront": "CLOSED",
    "doorDriverRear": "CLOSED",
    "doorLockState": "SECURED",
    "doorPassengerFront": "CLOSED",
    "doorPassengerRear": "CLOSED",
    "hood": "CLOSED",
    "internalDataTimeUTC": "2020-12-01T18:11:52",
    "lastChargingEndReason": "UNKNOWN",
    "lastChargingEndResult": "FAILED",
    "maxRangeElectric": 40,
    "maxRangeElectricMls": 24,
    "mileage": 148052,
    "parkingLight": "OFF",
    "position": {
      "status": "DRIVER_DISABLED"
    },
    "positionLight": "OFF",
    "rearWindow": "CLOSED",
    "remainingFuel": 14,
    "remainingRangeElectric": 16,
    "remainingRangeElectricMls": 9,
    "remainingRangeFuel": 194,
    "remainingRangeFuelMls": 120,
    "singleImmediateCharging": false,
    "trunk": "CLOSED",
    "updateReason": "VEHICLE_SHUTDOWN_SECURED",
    "updateTime": "2020-12-01T18:11:52+0000",
    "vin": "some_vin",
    "windowDriverFront": "CLOSED",
    "windowDriverRear": "CLOSED",
    "windowPassengerFront": "CLOSED",
    "windowPassengerRear": "CLOSED"
  }
}
  ```
</details>